projects/VitalRecord/VitalRecord_util.cs

```diff
@@ -388,6 +390,8 @@ protected Resource CreateEntry(FHIRPath fhirPath, string subjectId)
                 case FHIRPath.FhirType.Condition:
                     Condition condition = new Condition();
                     condition.Code = code;
+                    CodeableConcept usCoreCategory = new CodeableConcept(CodeSystems.ConditionCategory, "problem-list-item");
```

Comment:
Two of the conditions are profiles of a different USCore Condition (USCoreConditionEncounterDiagnosisProfile) with a different category (encounter-diagnosis)
- ConditionFetalDeathInitiatingCauseOrCondition
- ConditionFetalDeathOtherCauseOrCondition